### PR TITLE
do not duplicate MDC_KEY

### PR DESCRIPTION
**Pull request by bavardage**
This change is [Reviewable](https://reviewable.io/reviews/palantir/http-remoting/471)

### CODE REVIEW

**Review by gracew — APPROVED** [review]

File: tracing/src/main/java/com/palantir/remoting2/tracing/Tracers.java [hunk 1]
```diff
@@ -26,7 +26,7 @@
 /** Utility methods for making {@link ExecutorService} and {@link Runnable} instances tracing-aware. */
 public final class Tracers {
     /** The key under which trace ids are inserted into SLF4J {@link org.slf4j.MDC MDCs}. */
-    static final String MDC_KEY = "traceId";
+    public static final String MDC_KEY = "traceId";
```
Comment (uschi2000) [inline tracing/src/main/java/com/palantir/remoting2/tracing/Tracers.java]: I'm fine making this public, but can we choose a different name?

Comment (uschi2000) [reply tracing/src/main/java/com/palantir/remoting2/tracing/Tracers.java]: TRACE_ID_KEY or something like that

Comment (bavardage) [reply tracing/src/main/java/com/palantir/remoting2/tracing/Tracers.java]: done

**Comment by uschi2000** [comment]
I think we should merge this into the 3.0 branch.

**Pull request merged**